vpp/algorithms/Line_tracker_4_sfm/Hough_Extruder/paint.hh

```diff
@@ -0,0 +1,133 @@
+#pragma once
+
+#include <vpp/core/image2d.hh>
+#include <vpp/draw/draw.hh>
+#include <vpp/core/keypoint_trajectory.hh>
+#include <iod/sio_utils.hh>
+#include <vpp/algorithms/Line_tracker_4_sfm/miscellanous/operations.hh>
+#include <opencv2/highgui.hpp>
+#include "draw_line_hough.hh"
+#include "track.hh"
+
+using namespace vpp;
+using namespace cv;
+
+namespace vpp {
+
+
+void paint_hough_video(std::vector<track>& trs,
```

Comment:
This look like this is specific to a demo you made to display some results.
This will be nice to move it in the example dir and provide the demo that is using it,
so people can run it on their video themselves.
Otherwise, if you prefer not to, then you can just remove this function. 
In general, every function that is not called by another and that is not something that you want the end
user to use must be removed.